docs/hub/datasets-download-stats.md

```diff
@@ -0,0 +1,8 @@
+# Datasets Download Stats
+
+## How are download stats generated for datasets?
+
+The Hub provides download stats for all datasets loadable via the `datasets` library. To determine the number of downloads, the Hub counts every time `load_dataset` is called in Python, excluding Hugging Face's CI tooling on GitHub. This means that:
```

Comment:
I would maybe clarify that nothing is sent from the user's setup, it's just the server receiving a request to return a file that we count. 

Maybe a line such as 
> No information is sent from the user and no additional calls are made for this. The count is done server-side as we serve files for downloads.

Reply:
yes great point @LysandreJik